node/test/util/merge_util.js

```diff
@@ -378,6 +380,7 @@ a=B:Ca-1;Cb-1;Ba=a;Bb=b|
 x=S:C2-1 s=Sa:a;C3-1 s=Sa:b;Bmaster=2;Bfoo=3`,
                 fromCommit: "3",
                 expected: `x=E:Qmessage\n#M 2: 3: 0 3;I *s=~*S:a*S:b`,
+                fails: true,
```

Comment:
Why are these two tests now expected to fail?

Reply:
Originally errors are handled in two ways in: (1) `merge_utils` returns a object that contains `errorMessage` and `merge` command throws a `UserError` if `errorMessage` is not empty. and (2) `merge_utils` throws UserError directly. 

Now all errors from the `MergeUtil` module are thrown by `MergeUtil`. This test case was the case where `MergeUtil` returns an `errorMessage` but now throws an error. Hence it was considered as not fail before but actually it fails.